ddtrace/tracer/option.go

```diff
@@ -505,7 +506,12 @@ func newConfig(opts ...StartOption) (*config, error) {
 		c.agentURL = internal.AgentURLFromEnv()
 	}
 	c.originalAgentURL = c.agentURL // Preserve the original agent URL for logging
-	if c.httpClient == nil {
+	if c.httpClient == nil || orchestrion.Enabled() {
+		if orchestrion.Enabled() && c.httpClient != nil {
+			// Make sure we don't create http client traces from inside the tracer by using our http client
+			// TODO(eliott.bouhana): remove once dd:no-span is implemented
+			log.Error("Orchestrion is enabled, but a custom HTTP client was provided to tracer.Start. This is not supported and will be ignored.")
```

Comment:
```suggestion
			log.Debug("Orchestrion is enabled, but a custom HTTP client was provided to tracer.Start. This is not supported and will be ignored.")
```

Provided the experience on #2991, I'd rather to log this as `debug`. When attending escalation rotations, one of the usual steps is to activate debug logs.